packages/core/parcel-bundler/src/transforms/posthtml.js

```diff
@@ -38,15 +38,7 @@ async function getConfig(asset) {
   config = config || {};
   const plugins = config.plugins;
   if (typeof plugins === 'object') {
```

Comment:
Without `depConfig`, this conditional is redundant and should be removed.

Reply:
Should I drop the condition and call `Object.keys(plugins).forEach(p => plugins[p]);` directly?

Reply:
I suppose it would make sense to assume `plugins` is always an object.

Reply:
Sorry for the delayed response. Just to clarify, I should pull out the condition and the iteration of the plugins completely, right?

Reply:
Yes.